libgalois/include/galois/graphs/ReadGraph.h

```diff
@@ -41,27 +41,39 @@ void readGraph(GraphTy& graph, Args&&... args) {
 
 template <typename GraphTy>
 void readGraphDispatch(GraphTy& graph, read_default_graph_tag tag,
-                       const std::string& filename) {
+                       const std::string& filename,
+                       const bool readUnweighted = false) {
   FileGraph f;
-  f.fromFileInterleaved<typename GraphTy::file_edge_data_type>(filename);
-  readGraphDispatch(graph, tag, f);
+  if (readUnweighted) {
+    //! If user specifies that the input graph is unweighted,
```

Comment:
Doxygen syntax not required here; only required for defining vars and functions, not lines.

Reply:
Thank you for your comment. I'll keep it in my mind. 
But one thing that I want to mention is I summarized the comments cleaning as 'doxygen style', but also wanted to make consistent.